src/components/QRCode/index.js

```diff
@@ -38,6 +46,8 @@ const defaultProps = {
     color: defaultTheme.text,
     backgroundColor: defaultTheme.highlightBG,
     getRef: undefined,
+    logoRatio: 0.25,
+    logoMarginRatio: 0.02,
```

Comment:
can we add this numbers inside src/CONST.js file, it will be easier to track and change in one place, same for 0.22 and 0.03. Thanks

Reply:
I don't think extracting these values into CONST is a good idea, since we are not doing the same for size and in other places for default value.

Reply:
I think I agree with @narefyev91 on this one, especially since we re-use these figures in multiple places 🤔

Reply:
Extracted all values to `CONST.js`